packages/entrypoint/src/index.ts

```diff
@@ -15,8 +15,9 @@ declare const window: { [x: string]: any };
 
 // We would override __webpack_public_path__ only if __WPACKIO__ webpack plugin is set
 if (__WPACKIO__) {
+	const path = (__WPACKIO__.appName + __WPACKIO__.outputPath)
+		.toLowerCase()
+		.replace(/[^a-z0-9_-]/, '');
```

Comment:
This should be

```suggestion
		.replace(/[^a-z0-9_\-]/g, '');
```

Without this (`/g`), only the first instance of not accepted characters would be replaced. :)